docs/app/Examples/collections/Form/Types/FormFormExample.js

```diff
@@ -12,7 +12,7 @@ const FormFormExample = () => (
       <input placeholder='Last Name' />
     </Form.Field>
     <Form.Field>
-      <label>First Name</label>
+      <label>I agree to the Terms and Conditions</label>
```

Comment:
Let's delete this entirely since the Checkbox already has a label.

![image](https://cloud.githubusercontent.com/assets/5067638/19586829/14fdc0a4-970e-11e6-9502-0ce70636d6e6.png)